index.html

```diff
@@ -37,46 +37,192 @@
+          <nav>
+            <a href="#" id="hamburger"
+              ><i class="fa fa-bars" aria-hidden="true"></i
+            ></a>
+          </nav>
+        </div>
+        <ul class="menu_items">
+          <li class="close_button">
+            <img class="x_sign" src="./images/close_btn.svg" alt="Closing menu button" />
+          </li>
+          <li class="goto"><a href="#Portfolio">Portfolio</a><hr /></li>        
+          <li class="goto"><a href="#About">About</a><hr /></li>        
+          <li class="goto"><a href="#Contact">Contact</a><hr /></li>        
+        </ul>
+      </header>
+      <header class="desktop_toolbar">
+        <h3 class="logo">Micky's</h3>
+        <div class="desktop_options">
+          <h3>Portfolio</h3>
+          <h3>About</h3>
+          <h3>Contact</h3>
+          <h3><i class="fa fa-envelope" aria-hidden="true"></i></h3>
+        </div>
+      </header>
+      <section class="intro">
+        <div class="content">
+          <h1>Hey There.<br />I'm Michael</h1>
+          <h2>I’am a Software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+        <div class="social_icons">
+          <ul class="socials">
+            <li>
+              <a href=""><i class="fab fa-github"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-angellist"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-medium-m"></i></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="recent_works">
+        <div class="works_title">
+          <h3>My Recent Works</h3>
+          <hr />
+        </div>
+        <div class="headlines">
+          <div class="image_headline">
+            <img src="./images/Img_Placeholder.png" alt="Image place holder" />
+          </div>
+          <div class="discription_headline">
+            <h4>Multi-Post Stories</h4>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="tags">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Rubby</li>
+            </ul>
+            <button class="popup_display_button" id="1" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] I will like to commend the great job done here👏, kindly note that the mobile pop needs some adjustment to give a better UI(user interface). Kindly test the page with your browser dev tool across all mobile screens and see you are getting the desired result😎👍.
Your Implementation | Figma Design
![screenshot (9)](https://user-images.githubusercontent.com/44624138/151433726-7b65554d-4ddd-42d1-8d5e-320554b9ee71.png) | ![screenshot (8)](https://user-images.githubusercontent.com/44624138/151433806-affa27f5-00f4-444f-a432-2f897929de42.png)